app/src/src_nonfreenet/org/breezyweather/sources/bmd/BmdGeoLookup.kt

```diff
@@ -0,0 +1,647 @@
+/**
+ * This file is part of Breezy Weather.
+ *
+ * Breezy Weather is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation, version 3 of the License.
+ *
+ * Breezy Weather is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Breezy Weather. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.breezyweather.sources.bmd
+
+// This file contains Bengali names of divisions, districts, and upazilas used in reverse geolocation.
+// These names are hardcoded here rather than stored as resource strings,
+// as it serves little purpose to translate them into other languages.
+// Source: https://bmd.bdservers.site/src/js/language_array.js
```

Comment:
I would prefer them to be parsed from the web if they are available from there

Reply:
You can:
- Check the language first
- If `bn` => use the data
- Otherwise, in a try/catch:
   - Fetch that `.js` file
   - Parse it (regex + maybe with the Json parser to have it in a `Map`)
   - Try finding a match
   - In any case, fallback to the `bn` data

In any case, having it non-translated in the first version is fine

Reply:
Yeah, that's doable. That being said, I think I'll just remove the file for now so you can merge the rest first if they are good to go. Then add them back in a separate PR.